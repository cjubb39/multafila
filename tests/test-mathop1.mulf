/* math operation 1 */

int main (){
	int i = 4;
	int j = 5;
	
	j = (i + j) * i;
	
	printOut("should return 36");

	return j; /* should return 36 */

}