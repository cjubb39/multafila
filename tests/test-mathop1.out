
/*==========OUTPUT CODE BELOW==========*/
#include <stdio.h>
#include <pthread.h>
void printOut(char *m){printf("%s\n", m);}
void printInt(int m){printf("%d\n", m);}

int main (  );

pthread_mutex_t global_lock;

pthread_cond_t global_cond;

struct global_lock_struct_s{
} global_lock_struct;

pthread_t global_threads [0];
int main (  )
{

{
pthread_mutex_init( & global_lock , NULL);
pthread_cond_init( & global_cond , NULL);
}
 ;
i = 4 ;
j = 5 ;
j = (i + j) * i ;
printOut( "should return 36" ) ;
return j ;
}

